Cargo.toml

```diff
@@ -16,3 +16,4 @@ nix = "0.15"
 
 [features]
 vcan_tests = []
+can_fd = []
```

Comment:
I can understand why you put this here, I am not sure though whether it's not easier in long term to just leave out the feature and make it non optional. Might make sense if there are socketcan kernel versions that only support classical can though.

Reply:
I think this option does not complicate things. Leaving it optional means that users who do not use CAN FD at all will have those 16 B (+ tag) size structures instead of useless 72 B (+ tag). And as far I know most CAN controllers do not support CAN FD.

Reply:
I also think that it is overkill to have FD as a feature, let the user (or log file) tell whether FD should be enabled or not and risk wasting a few bytes if it isn't used. The differences between classical CAN and FD is so small anyway, so lets not complicate things more than necessary.